src/main/java/core/basesyntax/Application.java

```diff
@@ -1,7 +1,16 @@
 package core.basesyntax;
 
 public class Application {
+    private static final int NUMBER_OF_BALL = 3;
+
     public static void main(String[] args) {
         // create three balls using class Lottery and print information about them in console
+        Lottery lottery = new Lottery();
+        Ball[] randomBall = new Ball[NUMBER_OF_BALL];
```

Comment:
do we need this array?

Reply:
i think this array imtortate. i don't know how creat program whithout him

Reply:
we can write a cycle with one line `System.out.println(lottery.getRandomBall());`